Modules/_hashopenssl.c

```diff
@@ -329,24 +345,37 @@ EVP_repr(EVPobject *self)
     return PyUnicode_FromFormat("<%U HASH object @ %p>", self->name, self);
 }
 
-#if HASH_OBJ_CONSTRUCTOR
```

Comment:
So that the doc-string would still be generated, as it is used as the class's doc-string.

The init method is still not used, since its inclusion in the `PyTypeObject` definition is still conditional on `HASH_OBJ_CONSTRUCTOR`.